lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -124,6 +136,7 @@
                     *ngIf="col.draggable"
                     cdkDragHandle
                     [ngClass]="{ 'adf-datatable-cell-header-drag-icon': !isResizing }"
+                    aria-hidden="true"
```

Comment:
This should not be true always, there can be cases when this element is not hidden